platform/representation/src/gestures/DropGesture.js

```diff
@@ -165,16 +165,18 @@ define(
                     if (shouldCreateVirtualPanel(domainObject, selectedObject)){
                         editableDomainObject = createVirtualPanel(domainObject, selectedObject);
                         if (editableDomainObject) {
-                            navigationService.setNavigation(editableDomainObject);
+                            editableDomainObject.getCapability('action').perform('edit');
+                            //navigationService.setNavigation(editableDomainObject);
                             broadcastDrop(id, event);
-                            editableDomainObject.getCapability('status').set('editing', true);
+                            //editableDomainObject.getCapability('status').set('editing', true);
                         }
                     } else {
                         $q.when(action && action.perform()).then(function (result) {
                             //Don't go into edit mode for folders
                             if (domainObjectType!=='folder') {
-                                navigationService.setNavigation(editableDomainObject);
-                                editableDomainObject.getCapability('status').set('editing', true);
+                               // navigationService.setNavigation(editableDomainObject);
+                                //editableDomainObject.getCapability('status').set('editing', true);
+                                editableDomainObject.getCapability('action').perform('edit');
                             }
```

Comment:
Like the changes here to use actions. There are some commented-out lines that should just be removed.